app/assets/stylesheets/components/_intervention.scss

```diff
@@ -0,0 +1,5 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.gem-c-intervention {
+  margin-top: govuk-spacing(4);
+}
```

Comment:
We can remove this file if the approach mentioned in the show.html.erb view template is used